java/ddlogapi.c

```diff
@@ -229,29 +323,29 @@ JNIEXPORT jint JNICALL Java_ddlogapi_DDlogAPI_ddlog_1transaction_1commit_1dump_1
 
     jclass cls = (*env)->FindClass(env, "ddlogapi/DDlogAPI$FlatBufDescr");
     if (cls == NULL) {
-        return -1;
+        return;
```

Comment:
why not throw here?
this certainly looks like an error.

Reply:
Likewise, `FindClass` generates an exception on error, which will trigger as soon as we're back in the Java world.